tracing-filter/src/lib.rs

```diff
@@ -13,6 +13,8 @@ pub fn rust_log_add_quiet_defaults(
         Err(_) => RUST_LOG_SILENCE_DEFAULTS,
     };
 
+    println!("silence: {silence}");
```

Comment:
nit : `println!("rust logging silence: {silence}");`

Reply:
oops, that was debugging sorry, will be deleted!